sparql-ns.ttl

```diff
@@ -0,0 +1,403 @@
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-strstarts> ;
+    .
+
+sparql:strends rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-strends> ;
+    .
+
+sparql:contains rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-contains> ;
+    .
+
+sparql:strbefore rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-strbefore> ;
+    .
+
+sparql:strafter rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-strafter> ;
+    .
+
+sparql:concat rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-concat> ;
+    .
+
+sparql:langMatches rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-langMatches> ;
+    .
+
+sparql:regex rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-regex> ;
+    .
+
+sparql:replace rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-replace> ;
+    .
+
+sparql:encode rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-encode> ;
+    .
+
+# Section: Functions on numbers
+
+sparql:abs rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-abs> ;
+    .
+
+sparql:round rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-round> ;
+    .
+
+sparql:ceil rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-ceil> ;
+    .
+
+sparql:floor rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-floor> ;
+    .
+
+# Section: Functions on datetimes
+
+sparql:now rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-now> ;
+    .
+
+sparql:year rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-year> ;
+    .
+
+sparql:month rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-month> ;
+    .
+
+sparql:day rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-day> ;
+    .
+
+sparql:hours rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-hours> ;
+    .
+
+sparql:minutes rdf:type sparql:Function ;
+    dc:description "" ;
```

Comment:
This function extracts the minute component (0–59) from an `xsd:dateTime` value.

Reply:
I wonder why this includes a range, but the earlier similar functions do not?
```suggestion
    dc:description "This function returns the minute component (0–59) of an xsd:dateTime value." ;
```